channeldb/payments_test.go

```diff
@@ -426,6 +468,9 @@ func TestQueryPayments(t *testing.T) {
 					t.Fatalf("unable to create test "+
 						"payment: %v", err)
 				}
+				// Override creation time to allow for testing
+				// of MinCreationTimeNs and MaxCreationTimeNs.
```

Comment:
nit: The fields are being called differently now.

Reply:
fixed!